<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookmarkViewDialog</class>
 <widget class="QDialog" name="BookmarkViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View Bookmark</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout" stretch="0,1">
   <item>
    <widget class="QWidget" name="widTopPane" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>240</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>240</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
        <item>
         <widget class="QLabel" name="lblName">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel { font-size: 16pt; font-weight: bold; }</string>
          </property>
          <property name="text">
           <string>Bookmark Name</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="FiveStarRatingWidget" name="fvsRating" native="true"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widBookmarkProperties" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lblFolderLocation">
           <property name="text">
            <string>In folder &lt;span style=&quot;color:blue;&quot;&gt;%1&lt;/span&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="txtDesc">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="lblTags">
             <property name="text">
              <string>Tags:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="TagLineEdit" name="leTags">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QScrollArea" name="scrlBookmarkData">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrlBookmarkDataWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>308</width>
           <height>207</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="vlScrlBookmarkDataWidget" stretch="0,0,0,1">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="grpURLs">
            <property name="styleSheet">
             <string notr="true">QGroupBox { font-weight: bold; }</string>
            </property>
            <property name="title">
             <string>URLs</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="leftMargin">
              <number>6</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="QTextBrowser" name="txbURLs">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="lineWrapMode">
                <enum>QTextEdit::NoWrap</enum>
               </property>
               <property name="openExternalLinks">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="grpFiles">
            <property name="styleSheet">
             <string notr="true">QGroupBox { font-weight: bold; }</string>
            </property>
            <property name="title">
             <string>Attached Files</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="leftMargin">
              <number>6</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="QTableWidget" name="twAttachedFiles">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="verticalScrollMode">
                <enum>QAbstractItemView::ScrollPerPixel</enum>
               </property>
               <property name="horizontalScrollMode">
                <enum>QAbstractItemView::ScrollPerPixel</enum>
               </property>
               <attribute name="horizontalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="grpRelatedBookmarks">
            <property name="styleSheet">
             <string notr="true">QGroupBox { font-weight: bold; }</string>
            </property>
            <property name="title">
             <string>Related Bookmarks</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="leftMargin">
              <number>6</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="BookmarksView" name="bvLinkedBookmarks" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="vlSqueezeTheOtherThree">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="FilePreviewerWidget" name="widPreviewer" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BookmarksView</class>
   <extends>QWidget</extends>
   <header>Bookmarks/BookmarksView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TagLineEdit</class>
   <extends>QLineEdit</extends>
   <header>Tags/TagLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>FiveStarRatingWidget</class>
   <extends>QWidget</extends>
   <header>Bookmarks/FiveStarRatingWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FilePreviewerWidget</class>
   <extends>QWidget</extends>
   <header>FileViewer/FilePreviewerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtDesc</tabstop>
  <tabstop>leTags</tabstop>
  <tabstop>scrlBookmarkData</tabstop>
  <tabstop>txbURLs</tabstop>
  <tabstop>twAttachedFiles</tabstop>
  <tabstop>bvLinkedBookmarks</tabstop>
  <tabstop>widPreviewer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
